src/CMake/FindVisItPython.cmake

```diff
@@ -421,125 +423,120 @@ IF(VISIT_PYTHON_SKIP_INSTALL)
-        IF((NOT ${PYTHON_DIR} STREQUAL "/usr"))
-            # Install the python executable
-            STRING(SUBSTRING ${PYTHON_VERSION} 0 1 PYX)
-            STRING(SUBSTRING ${PYTHON_VERSION} 0 3 PYX_X)
-            THIRD_PARTY_INSTALL_EXECUTABLE(${PYTHON_DIR}/bin/python ${PYTHON_DIR}/bin/python${PYX} ${PYTHON_DIR}/bin/python${PYX_X})
-
-            # Install the python modules
-            # Exclude lib-tk files for now because the permissions are bad on davinci. BJW 12/17/2009
-            # Exclude visit module files.
-            IF(EXISTS ${PYTHON_DIR}/lib/python${PYTHON_VERSION})
-                INSTALL(DIRECTORY ${PYTHON_DIR}/lib/python${PYTHON_VERSION}
-                    DESTINATION ${VISIT_INSTALLED_VERSION_LIB}/python/lib
-                    FILE_PERMISSIONS OWNER_READ OWNER_WRITE
-                                     GROUP_READ GROUP_WRITE
-                                     WORLD_READ
-                    DIRECTORY_PERMISSIONS OWNER_READ OWNER_WRITE OWNER_EXECUTE
-                                          GROUP_READ GROUP_WRITE GROUP_EXECUTE
-                                          WORLD_READ WORLD_EXECUTE
-                    PATTERN "lib-tk" EXCLUDE
-                    PATTERN "visit.*" EXCLUDE
-                    PATTERN "visitmodule.*" EXCLUDE
-                    PATTERN "visit_writer.*" EXCLUDE
-                    PATTERN "PySide" EXCLUDE
-                    PATTERN "Python-2.6-py2.6.egg-info" EXCLUDE
-                )
-            ENDIF(EXISTS ${PYTHON_DIR}/lib/python${PYTHON_VERSION})
-
-            # Install the Python headers
-            IF (NOT WIN32)
-                IF(VISIT_HEADERS_SKIP_INSTALL)
-                    MESSAGE(STATUS "Skipping python headers installation")
-                ELSE(VISIT_HEADERS_SKIP_INSTALL)
-                    INSTALL(DIRECTORY ${PYTHON_INCLUDE_PATH}
-                        DESTINATION ${VISIT_INSTALLED_VERSION_INCLUDE}/python/include
-                        FILE_PERMISSIONS OWNER_READ OWNER_WRITE
-                                         GROUP_READ GROUP_WRITE
-                                         WORLD_READ
-                        DIRECTORY_PERMISSIONS OWNER_READ OWNER_WRITE OWNER_EXECUTE
-                                              GROUP_READ GROUP_WRITE GROUP_EXECUTE
-                                              WORLD_READ             WORLD_EXECUTE
-                        PATTERN ".svn" EXCLUDE
-                        )
-                ENDIF(VISIT_HEADERS_SKIP_INSTALL)
-                #
-                # CDH:
-                # We also need to install the headers into lib/python dir
-                # if we want to be able to install python modules into a visit
-                # install.
-                #
+    MESSAGE(STATUS "We will install Python along with VisIt")
+    # Install libpython
+    # split the list so that dll's are found correctly on Windows
+    foreach(pylib ${PYTHON_LIBRARIES})
```

Comment:
Its a small nit but inconsistent CMake command casing here.

Reply:
I believe we are trying to switch to the lower case versions in new code since that is the new style for doing CMake. Therefore, I think this is ok. Maybe at some point we should try to switch to lower case throughout.

Reply:
Wouldn't it be best to avoid *mixing* case in the same file though?

Reply:
I believe that's what we've been doing. The alternative is to change every reference in the file and that creates noise when looking at changes. It seems that file wide case changes should be there own commits. I don't have a strong opinion here, just mentioning what we've been doing. Perhaps others can chime in.

Reply:
No. Thats fine. I rescind my remark there. I agree that changing the whole file's CMake command case in addition to change the CMake logic would be a bad way to go 😞